tos/chips/rf233/RF230DriverHwAckP.nc

```diff
@@ -588,13 +613,15 @@ tasklet_async command uint8_t RadioState.getChannel()
 		// we have missed an incoming message in this short amount of time
 		if( (readRegister(RF230_TRX_STATUS) & RF230_TRX_STATUS_MASK) != RF230_TX_ARET_ON )
 		{
-			RADIO_ASSERT( (readRegister(RF230_TRX_STATUS) & RF230_TRX_STATUS_MASK) == RF230_BUSY_RX_AACK );
+			RADIO_ASSERT( (readRegister(RF230_TRX_STATUS) & RF230_TRX_STATUS_MASK) == RF230_BUSY_RX_AACK );\
```

Comment:
I must have put it there as a mistake. Somehow the code still compiles fine with the backslash. Will push a fix.